pkg/lifecycle/lifecycle.go

```diff
@@ -23,48 +25,19 @@ type Asset struct {
 // assets versions map, version rules and methods to apply the lifecycle rules in the target branch
 type Dependencies struct {
 	RootFs            billy.Filesystem
-	assetsVersionsMap map[string][]Asset
+	AssetsVersionsMap map[string][]Asset
 	VR                *VersionRules
 	Git               *git.Git
-	// These wrappers are used to mock the filesystem and git status in the tests
-	walkDirWrapper         WalkDirFunc
-	makeRemoveWrapper      MakeRemoveFunc
-	statusPorceLainWrapper StatusPorcelainFunc
-	addAndCommitWrapper    AddAndCommitFunc
+	walkDirWrapper    WalkDirFunc // Used to mock the filesystem in unit-tests
```

Comment:
This is fine since it's in this repo but generally code for testing should never exist in the implementation code.